c2corg_ui/static/js/alertservice.js

```diff
@@ -0,0 +1,54 @@
+goog.provide('app.Alerts');
+
+
+goog.require('app');
+
+
+
+/**
+ * @constructor
+ * @export
+ */
+app.Alerts = function() {
+
+  /**
+   * @type {Array.<appx.AlertMessage>}
+   * @private
+   */
+  this.alerts_ = [];
+};
+
+
+/**
+ * @param {string|appx.AlertMessage} msg Alert data.
```

Comment:
Yes, or I could force using an ``appx.AlertMessage`` object since its ``type``and ``timeout`` attributes are optional anyway:
```
this.alerts_.add({'msg': msg});
```